### ISSUE TITLE
Not able to connect to wifi in android 10 .

### ISSUE THREAD

**Issue by MahendraDhaker** [steinwurf/adb-join-wifi#15]
![image](https://user-images.githubusercontent.com/31129084/210348280-114fa0e4-dfdb-46f4-94e1-2d75a3fab739.png)
![image](https://user-images.githubusercontent.com/31129084/210348382-affd515a-a6d1-487b-bad6-2242fc8760ca.png)

**Comment by veitalmaphil** [comment]
yes, this project is 5 years old and no longer useful unfortunately
